04-ruby-mvc-sinatra/active-record-intro/readme.md

```diff
@@ -158,6 +163,9 @@ gem "pry"  # this gem allows access to REPL
 Then I'm going to run `$ bundle install` in the terminal.
 
 ### Setup Ruby - Tunr (You Do - 10 / 70)
+If you did not complete the initial SQL setup for Tunr above:
```

Comment:
`git fetch https://github.com/ga-dc/tunr_sinatra` first to make sure all remote branches are fetched.

then `git checkout 2_active_record_starter`